### PR TITLE
Fix log message

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

File: lib/vsc/filesystem/posix.py [hunk 1]
```diff
@@ -383,7 +383,7 @@ def populate_home_dir(self, user_id, group_id, home_dir, ssh_public_keys):
         @type ssh_public_keys: list of strings representing the public ssh keys
         """
         # ssh
-        self.log.info("Populating home %s for user %s:%s" % (home_dir, user_id, group_id))
+        self.log.info("Populating home %s for user %s:%s", home_dir, user_id, group_id)
```
Comment (stdweird) [inline lib/vsc/filesystem/posix.py]: what does it fix?

File: setup.py [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
 
 PACKAGE = {
-    'version': '0.36',
+    'version': '0.36.1',
```
Comment (stdweird) [inline setup.py]: i don't like to switch numbering scheme. please use `0.37`

File: lib/vsc/filesystem/posix.py (hunk 1 above)
Comment (itkovian) [reply lib/vsc/filesystem/posix.py]: Now that I look at it, that is a good question. The error was not enough values for interpolation.

File: setup.py (hunk 2 above)
Comment (itkovian) [reply setup.py]: No. We use patch level numbers everywhere else as well. This does not warrant a minor version update at all.

File: lib/vsc/filesystem/posix.py (hunk 1 above)
Comment (itkovian) [reply lib/vsc/filesystem/posix.py]: But in any case, this the better format :)

File: setup.py (hunk 2 above)
Comment (stdweird) [reply setup.py]: yes, but we weren't here before, so no reason to start now.

**Comment by itkovian** [comment]
I think I misread something. Closing until further notice.

**Pull request closed without merging**